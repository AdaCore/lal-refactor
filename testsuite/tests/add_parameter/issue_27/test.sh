add_parameter -P default.gpr -S main_package.ads -L 3 -R 6 -N 'Success : out Boolean'
