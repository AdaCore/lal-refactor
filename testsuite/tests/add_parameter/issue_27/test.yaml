description: |+
    The that the Add Parameter tool correctly adds a parameter to a function
    without parameters.

driver: shell_script
