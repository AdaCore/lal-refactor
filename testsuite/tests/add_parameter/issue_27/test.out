Adding parameter Success : out Boolean
main_package.adb
3:27-3:27  (Success : out Boolean)
main_package.ads
3:27-3:27  (Success : out Boolean)

